lib/ansible/modules/network/junos/_junos_vlan.py

```diff
@@ -79,31 +79,30 @@
 EXAMPLES = """
 - name: configure VLAN ID and name
   junos_vlan:
-    vlan_name: test
+    name: test
     vlan_id: 20
-    name: test-vlan
 
 - name: Link to logical layer 3 interface
   junos_vlan:
-    vlan_name: test
+    name: test
     vlan_id: 20
     l3-interface: vlan.20
     name: test-vlan
```

Comment:
Not sure what this line was originally for, but one of these names is unnecessary

Reply:
Already done ;)